### PR TITLE
Puppetry

### PR DESCRIPTION
**Pull request by dsheiko**
https://github.com/dsheiko/puppetry has now 400 stars and ~ 10 000 downloads, including quite many of Mac version https://www.somsubhra.com/github-release-stats/?username=dsheiko&repository=puppetry

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/puppetry.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  # github.com/dsheiko/puppetry was verified as official when first introduced to the cask
+  url "https://github.com/dsheiko/puppetry/releases/download/v#{version}/puppetry-mac-v#{version}.zip"
+  appcast 'https://github.com/dsheiko/puppetry/releases.atom'
+  name 'puppetry'
+  homepage 'https://puppetry.app/'
+
+  app 'puppetry.app'
+
+  uninstall signal: [
```
Comment (vitorgalvao) [inline Casks/puppetry.rb]: Isn’t `quit:` enough?

**Comment by vitorgalvao** [comment]
You’re removing another cask in the PR.

**Comment by dsheiko** [comment]
"You’re removing another cask in the PR."
I believe I did pull the repo before  pushing.. Gonna check again

**Comment by dsheiko** [comment]
Merged the upstream repository into my fork

**Comment by vitorgalvao** [comment]
You’re still deleting another cask, and have several extraneous commits. I recommend you try from the beginning.

**Pull request closed without merging**